Sources/Widgets/Representations/ConeHandleRepresentation/index.js

```diff
@@ -0,0 +1,160 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkHandleRepresentation from 'vtk.js/Sources/Widgets/Representations/HandleRepresentation';
+import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+import vtkPixelSpaceCallbackMapper from 'vtk.js/Sources/Rendering/Core/PixelSpaceCallbackMapper';
+import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
+import vtkGlyph3DMapper from 'vtk.js/Sources/Rendering/Core/Glyph3DMapper';
+import vtkConeSource from 'vtk.js/Sources/Filters/Sources/ConeSource';
+import { ScalarMode } from 'vtk.js/Sources/Rendering/Core/Mapper/Constants';
```

Comment:
For more readibility, you can order your vtk.js imports in alphabetic order

Reply:
I think you forgot to update these imports.

Reply:
Can you reorder in alphabetic order ?

Reply:
`import macro from 'vtk.js/Sources/macro';
import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
import vtkConeSource from 'vtk.js/Sources/Filters/Sources/ConeSource';
import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
import vtkGlyph3DMapper from 'vtk.js/Sources/Rendering/Core/Glyph3DMapper';
import vtkHandleRepresentation from 'vtk.js/Sources/Widgets/Representations/HandleRepresentation';
import vtkPixelSpaceCallbackMapper from 'vtk.js/Sources/Rendering/Core/PixelSpaceCallbackMapper';
import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
import { ScalarMode } from 'vtk.js/Sources/Rendering/Core/Mapper/Constants';`

Reply:
Doesn't seem to be done.

Reply:
Please reorder.

Reply:
Please reorder.